recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/camerabe/camerabe_git.bbappend

```diff
@@ -0,0 +1,10 @@
+################################################################################
+# Renesas R-Car
+################################################################################
+SRCREV_rcar = "${AUTOREV}"
+
+SRC_URI_append_rcar = " git://github.com/xen-troops/camera_be.git;protocol=https;branch=master"
+
+EXTRA_OECMAKE_append_rcar = " -DWITH_DOC=OFF"
+
+RDEPENDS_${PN} = "libxenbe libconfig"
```

Comment:
v4l2 dependencies?

Reply:
Linrary "libv4l2" was added to dependencies. Library name was taken from makefiles of camera_be.